File: ViewPumpSample/src/main/java/dev/b3nedikt/viewpump/sample/ViewPumpApplication.java
package dev.b3nedikt.viewpump.sample;

import android.app.Application;

import dev.b3nedikt.viewpump.ViewPump;

/**
 * For ViewPump.
 */
public class ViewPumpApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        ViewPump.init(new TextUpdatingInterceptor(), new CustomTextViewInterceptor());
    }
}

File: app/src/main/java/dev/b3nedikt/viewpump/sample/ViewPumpApplication.kt
package dev.b3nedikt.viewpump.sample

import android.app.Application
import dev.b3nedikt.app_locale.AppLocale
import dev.b3nedikt.app_locale.SharedPrefsAppLocaleRepository
import dev.b3nedikt.viewpump.ViewPump

/**
 * For ViewPump.
 */
class ViewPumpApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        AppLocale.appLocaleRepository = SharedPrefsAppLocaleRepository(this)

        ViewPump.init(TextUpdatingInterceptor(), CustomTextViewInterceptor())
    }
}